src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -293,6 +295,20 @@ private static void syncConfig(boolean load)
         replaceVanillaBucketModel = prop.getBoolean(Boolean.FALSE);
         propOrder.add(prop.getName());
 
+        propOrder = new ArrayList<String>();
```

Comment:
I wasn't sure about configuration being a part of it, as it's more of an appearance related tweak it may not appeal to everyone. That said it could mean missing out on some information that's intended to be conveyed.